.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,10 +4,10 @@
 
 * [ ] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
 * [ ] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
-* [ ] Have you picked the single best (existing) category for your project?
-* [ ] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
+* [ ] Have you picked the single best (existing) category and subcategory for your project?
+* [ ] Does it follow the other guidelines from the [new entries](https://github.com/NREL/mobility_landscape#new-entries) section?
 * [ ] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
-* [ ] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
+* [ ] Does your logo follow the [guidelines](https://github.com/NREL/mobility_landscape#logos)?
 * [ ] Does your project/product name match the text on the logo?
-* [ ] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
+* [ ] Have you ensured that all of your filter variables meet the standards of other entries?
 * [ ] ~15 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?
```

Comment:
You also need to change this to describe the process of looking up the artifact from the checks. Since we don't use netlify, there is no URL for the staging server. People just need to check the landscape png and pdf files to verify that everything is correct.

This is why I walked you through the process.